src/components/core/header.js

```diff
@@ -90,20 +91,23 @@ const activeLinkStyle = color => ({
 const MenuElement = ({ label, href, slug, color, ...props }) => (
   <MenuElementWrapper {...props}>
     {href ? (
-      <a href={href} css={linkStyle} target="_blank" rel="noopener noreferrer">
+      <a href={href} css={linkStyle} onClick={() => logAmplitudeEvent('Link to Developer Console clicked', {})} target="_blank" rel="noopener noreferrer">
         {label}
       </a>
     ) : (
-      <Link
-        to={`/${slug}/`}
-        partiallyActive
-        css={linkStyle}
-        activeStyle={activeLinkStyle(color)}
-      >
-        <CategoryIcon category={slug} style={iconStyle} />
-        {label}
-      </Link>
-    )}
+        <Link
+          to={`/${slug}/`}
+          partiallyActive
+          css={linkStyle}
+          activeStyle={activeLinkStyle(color)}
+          onClick={() => logAmplitudeEvent('Tab selected from top menu', {
+            slug: slug
```

Comment:
```suggestion
            slug
```

You can use object shorthand notation